### PR TITLE
Gitea update to 1.15.10

### PR DESCRIPTION
**Pull request by Wuszek**
## Description

Upgrade package to 1.15.10

## Checklist

- [x] Build rule `all-supported` completed successfully
- [ ] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [x] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Update-Policy#tests-checks)
- [x] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [ ] Bug fix
- [ ] New Package
- [x] Package update
- [ ] Includes small framework changes
- [ ] This change requires a documentation update (e.g. Wiki)

### CODE REVIEW

File: spk/gitea/Makefile [hunk 1]
```diff
@@ -1,12 +1,12 @@
 SPK_NAME = gitea
-SPK_VERS = 1.15.9
+SPK_VERS = 1.15.10
 SPK_REV = 1
```
Comment (hgy59) [inline spk/gitea/Makefile:3]: each package update must increment the SPK_REV
```suggestion
SPK_REV = 2
```

**Comment by hgy59** [comment]
we should first fix the ssh issue with #5071, before publishing an update.

File: spk/gitea/Makefile (hunk 1 above)
Comment (Wuszek) [reply spk/gitea/Makefile]: Done

**Comment by Wuszek** [comment]
> we should first fix the ssh issue with #5071, before publishing an update.

Sure, update can wait. Gitea just got v1.16.0-rc1 release, so maybe I will just update package to next release, after SSH fix.

**Comment by Wuszek** [comment]
@publicarray already did update to 1.16.0 along with some fixes here https://github.com/SynoCommunity/spksrc/pull/5071

**Pull request closed without merging**